packages/chain-helpers/package.json

```diff
@@ -2,15 +2,24 @@
   "name": "@kiltprotocol/chain-helpers",
   "version": "0.25.0-2",
   "description": "",
-  "main": "./lib/index.js",
-  "typings": "./lib/index.d.ts",
+  "main": "./lib/cjs/index.js",
+  "module": "./lib/esm/index.js",
+  "typings": "./lib/cjs/index.d.ts",
```

Comment:
I believe the "default" is actually `types`, i.e. https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html#including-declarations-in-your-npm-package

Reply:
d67ed8874d8a0b97c72a90ad9762f373ee0aa6e8